.github/workflows/main.yaml

```diff
@@ -2,87 +2,121 @@ name: Build, push and deploy
 on: push
 env:
   IMAGE_TAG: ${{ github.sha }}
-  IMAGE: docker.pkg.github.com/${{ github.repository }}/veilarbperson
+  IMAGE: ghcr.io/${{ github.repository }}/veilarbperson
   PRINT_PAYLOAD: true
 jobs:
+  link-to-trello:
+    runs-on: ubuntu-latest
+    name: Trello update
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get trello card id
+        id: card
+        uses: navikt/veilarbtrelloactions/get-card@v2.0
+        with:
+          key: ${{ secrets.TRELLO_KEY }}
+          token: ${{ secrets.TRELLO_TOKEN }}
+          board: BcjrjQZk
+      - name: Attach branch to card
+        id: Attach
+        uses: navikt/veilarbtrelloactions/create-attachment@v2.0
+        if: ${{ steps.card.outputs.success }}
+        with:
+          key: ${{ secrets.TRELLO_KEY }}
+          token: ${{ secrets.TRELLO_TOKEN }}
+          card-id: ${{ steps.card.outputs.card-id }}
+          attachment-type: branch
   test:
     name: Run tests
     runs-on: ubuntu-latest
-    if: github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/master'
```

Comment:
No test in dev? 
i think I disagree

Reply:
Will fix..

Reply:
Although its the same in veilarbportefolje..

Reply:
Tests will run in dev... condition is `github.ref != 'refs/heads/master'`